client/src/components/friend.jsx

```diff
@@ -2,35 +2,26 @@ import { Flex, Button } from '@chakra-ui/react';
 import React, { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import axios from 'axios';
-import { auth } from '../firebase';
-import { useAuthState } from 'react-firebase-hooks/auth';
-import pepe from '../../../public/images/PepeProfile.jpeg';
+import pepe from '../../public/images/PepeProfile.jpeg';
 
 function Friend({ user_id, friend, event_id }) {
   const navigate = useNavigate();
   const [isFriend, setIsFriend] = useState(friend.friend);
   const [isInvited, setIsInvited] = useState(false);
-  const [user, loading, error] = useAuthState(auth);
 
   useEffect(() => {
-    axios.get(`/api/events/users/${event_id}`)
-      .then((res) => {
-        res.data.forEach((user) => {
-          if (user.user_id === friend.id) { setIsInvited(true); }
-        });
+    axios.get(`/api/events/users/${event_id}`).then((res) => {
+      res.data.forEach((user) => {
+        if (user.user_id === friend.id) {
+          setIsInvited(true);
+        }
       });
+    });
   }, [friend, event_id]);
 
-  useEffect(() => {
```

Comment:
Eric confirmed that we can remove this since the friends menu isn't shown when the user is logged out